Online-Messaging-App/src/app/home/header/header.component.html

```diff
@@ -3,32 +3,35 @@
 
```

Comment:
please right click on the src folder of Online-Messaging-App and click "Reformat with Prettier" before you push up again

Reply:
done